tests/robottelo/decorators/test_host.py

```diff
@@ -0,0 +1,166 @@
+import six
```

Comment:
Add docstrings to the module and methods showing what are the goals of each test, that will give a brief idea of what the test is trying to accomplish and will help others knowing that when reading the code.